ibm_catalog.json

```diff
@@ -202,7 +198,93 @@
             {
               "key": "skip_iam_authorization_policy"
             }
-          ]
+          ],
+          "dependencies": [
+            {
+              "name": "deploy-arch-ibm-account-infra-base",
+              "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+              "flavors": ["standard"],
+              "id": "63641cec-6093-4b4f-b7b0-98d2f4185cd6-global",
+              "ignore_auto_referencing": ["*"],
+              "input_mapping": [
+                {
+                  "dependency_output": "workload_resource_group_name",
```

Comment:
this should have `"reference_version": true` so the user can set the name at the top